src/cmd_argument.rs

```diff
@@ -47,6 +47,9 @@ pub struct Split<'a>(pub &'a str);
 ///
 /// let StdoutTrimmed(output) = cmd!(Split("echo foo"));
 /// assert_eq!(output, "foo");
+///
+/// let StdoutTrimmed(output) = cmd!(Split(format!("echo {}", 100)));
+/// assert_eq!(output, "100");
```

Comment:
Maybe we should also have a test in `lib.rs`? So far I'm using doctests in this project more for (automatically checked) documentation, and not for test coverage.

Reply:
Sounds good! I'll add a real test.

Reply:
Thank you!